.github/workflows/help-full-command.yml

```diff
@@ -0,0 +1,60 @@
+name: Run help-full command for e2e-testing-tool
```

Comment:
Can we consider another name? **help-full** tells me it will help with anything but it works only with testing-tool.

Reply:
Unfortunately, no, we cannot rename help and help-full actions for testing tools. We need to pass the entire comment to the testing tool along with the command and slash.

Reply:
Although, I have one idea)

Reply:
I propose to plan this renaming because it's all over the place, including screenshots in the manual. This change makes sense, but it's really painful because we are in the middle of developing core features.

Reply:
@cuyk, @DoNotPanicUA, I suggest leaving it as it is for testing these commands. While these teams are being tested, let's discuss how best to change it.